### DISCUSSION TITLE
ValidationError: Graph must be in single static assignment (SSA) form, however 'layer1_0_relu0_fwd' has been used as output names multiple times.

### DISCUSSION THREAD

**Discussion by aslam-ep** [apache/incubator-mxnet#20205] [General]
When converting MXNet i3d_resnet50_v1_custom model to onxx I got an error like this:

`ValidationError: Graph must be in single static assignment (SSA) form, however 'layer1_0_relu0_fwd' has been used as output names multiple times.`

onxx version I used: 1.2.1

**Comment by Zha0q1** [comment]
hi @aslam-ep 
This issue has been fixed in v1.x branch. To access the new onnx support you can refer to this instruction https://github.com/apache/incubator-mxnet/tree/v1.x/python/mxnet/onnx#installation. You can build a wheel for mx2onnx and continue to use your current mxnet. 

BTW Do you have any specific reason to use onnx 1.2.1? I would recommend using onnx 1.7 or 1.8 because that's what we officially support now

**Reply by Zha0q1** [reply]
```
##Installation

From the 1.9 release and on, the ONNX export module has become an offical, built-in module in MXNet. You can access the module at mxnet.onnx.

If you are a user of earlier MXNet versions and do not want to upgrade MXNet, you can still enjoy the latest ONNX suppor by pulling the MXNet source code and building the wheel for only the mx2onnx module. Just do cd python/mxnet/onnx and then build the wheel with python3 -m build. You should be able to find the wheel under python/mxnet/onnx/dist/mx2onnx-0.0.0-py3-none-any.whl and install it with pip install mx2onnx-0.0.0-py3-none-any.whl. You should be able to access the module with import mx2onnx then.
```

**Reply by aslam-ep** [reply]
Could you be more specific I am a newb and I don't how to build a wheel

**Reply by aslam-ep** [reply]
Thanks, man it worked.

**Reply by Zha0q1** [reply]
glad I was able to help!

**Reply by gospodima** [reply]
Hello @Zha0q1 
I am having similar problem (I use mxnet 1.7.0), but I also need onnx2mx functionality. Is it somehow possible to use import&export functions with onnx 1.7 support from mxnet 1.7.0?
